crates/indexer/src/get_storefronts.rs

```diff
@@ -129,6 +138,8 @@ fn process_tags(
         let logo_url = tags
             .remove("holaplex:theme:logo:url")
             .unwrap_or_else(String::new);
+        
+        let created_at = Some(timestamp).map(|t| NaiveDateTime::from_timestamp(t, 0));
```

Comment:
```suggestion
        let created_at = Some(NaiveDateTime::from_timestamp(timestamp, 0));
```

Reply:
`Some(...).map(...)` is redundant, you can just operate on `timestamp` directly and wrap the whole thing in an `Option` afterwards